R/expr-expr.R

```diff
@@ -2311,15 +2311,22 @@ expr__hist <- function(
 #' df <- df$select(pl$col("color")$value_counts(sort = TRUE, name = "n"))
 #' df
 #'
-#' df$unnest()
+#' df$unnest("n")
 expr__value_counts <- function(
     ...,
     sort = FALSE,
     parallel = FALSE,
-    name = "count",
+    name = NULL,
     normalize = FALSE) {
   wrap({
     check_dots_empty0(...)
+    if (is.null(name)) {
+      if (isTRUE(normalize)) {
+        name <- "proportion"
+      } else {
+        name <- "count"
+      }
+    }
```

Comment:
roxygen comments also need to be updated?

Reply:
Yes, good catch. Done